src/components/GoalScreenHeader/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+import Images from '../../constants/images/Image';
+import { GoalScreenHeaderStyle } from './GoalScreenHeaderStyle';
+
+export type GoalScreenHeaderProps = {
+  title: String;
+  onPress: Function;
+  shouldShowArrowDownIcon: Boolean;
+};
+
+function GoalScreenHeader(props: GoalScreenHeaderProps) {
```

Comment:
The name could be changed from goalScreenheader to longTermSectionHeader

Reply:
I have keep it as goalScreenHeader as  name to make it a reusable component common to any other component that may be introduced in the future. as it accepts name ,  callback on other function as props.

Reply:
ok got it.